logstash-input-stdin.gemspec

```diff
@@ -23,7 +23,6 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
   s.add_runtime_dependency "logstash-codec-line"
   s.add_runtime_dependency "concurrent-ruby"
-  s.add_runtime_dependency "jruby-stdin-channel"
```

Comment:
Do we know precisely which version of Logstash and/or Jruby is needed in order to preclude the need for `jruby-stdin-channel`? If so we should add that as a requirement so that this update isn't pulled in on Logstashes/Jrubies that still need it.

Reply:
:+1: believe the changes are due to the JRuby 9K IO sub-system rewrite.
so any 9.x should do, either `logstash-core >= 6.0` or simply `s.required_ruby_version '>= 2.3'`

Reply:
> There is still an open issue on jruby about stdin's interruptibility, noting that support may have gotten better on linux/darwin, but that there may still be issues with Windows: jruby/jruby#2024

missed this one, I think it's still an issue - even on Linux/Mac. 
we might need a spec checking for `in.read` followed by an `in.close` not causing a hang (till a char is received).